### DISCUSSION TITLE
[Feature] Provide a script to automate breaking change fixes

### DISCUSSION THREAD

**Discussion by paaspaas00** [immich-app/immich#10222] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Your breaking change notes are awesome! 
Since they're so detailed, you could simply, with minimal added effort, provide a script to automate them, and that's run when updating incrementally.
It would be very nice to have!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Thanks for your request. I don't believe this is something that we will implement. There are far too many deployment setups to cover in a script and it's better for the administrator to understand the changes being made to avoid errors from taking place.
